src/Services/FHIR/FhirPatientService.php

```diff
@@ -2,14 +2,26 @@
 
 namespace OpenEMR\Services\FHIR;
 
+use Ramsey\Uuid\Uuid;
+use OpenEMR\Services\FHIR\iFhirService;
 use OpenEMR\Services\PatientService;
 use OpenEMR\FHIR\R4\FHIRDomainResource\FHIRPatient;
 use OpenEMR\FHIR\R4\FHIRElement\FHIRAddress;
 use OpenEMR\FHIR\R4\FHIRElement\FHIRHumanName;
 use OpenEMR\FHIR\R4\FHIRElement\FHIRAdministrativeGender;
 use OpenEMR\FHIR\R4\FHIRElement\FHIRId;
 
-class FhirPatientService
+/**
+ * FHIR Patient Service Tests
+ * @coversDefaultClass OpenEMR\Services\FHIR\FhirPatientService
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2020 Jerry Padgett <sjpadgett@gmail.com>
```

Comment:
This doesn't seem right? Also don't forget to include yourself in authorship. This helps us know who to blame if something goes wrong!
Again, kidding:)

Reply:
Ha! Will do.